### PR TITLE
[Doc] address getting-started-on-prem document issues [skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
This pr is to address #8280 to fix getting-started-on-prem document issues.

Signed-off-by: liyuan <yuali@nvidia.com>

### CODE REVIEW

File: docs/get-started/getting-started-on-prem.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ parent: Getting-Started
 ---
 
 # Getting Started with RAPIDS Accelerator with on premise cluster or local mode
-## Spark Deployment Methods
+## The primary methods of deploy Spark
 The way you decide to deploy Spark affects the steps you must take to install and setup Spark and
 the RAPIDS Accelerator for Apache Spark. The primary methods of deploy Spark are:
```
Comment (sameerz) [inline docs/get-started/getting-started-on-prem.md:11]:
```suggestion
the RAPIDS Accelerator for Apache Spark. The primary methods to deploy Spark are:
```

File: docs/get-started/getting-started-on-prem.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ parent: Getting-Started
 ---
 
 # Getting Started with RAPIDS Accelerator with on premise cluster or local mode
-## Spark Deployment Methods
+## The primary methods of deploy Spark
```
Comment (sameerz) [inline docs/get-started/getting-started-on-prem.md:9]: Although I know this was suggested in issue https://github.com/NVIDIA/spark-rapids/issues/8280 , I think the original phrasing is ok
```suggestion
## Spark Deployment Methods
```

**Review by mythrocks — APPROVED** [review]
A couple of minor nitpicks, but LGTM.

**Review by mythrocks — APPROVED** [review]

**Comment by nvliyuan** [comment]
build

**Pull request merged**